dev-tools/packaging/templates/ironbank/Dockerfile.tmpl

```
@@ -75,8 +75,19 @@ RUN find / -xdev -perm -4000 -exec chmod u-s {} +
 RUN groupadd --gid 1000 ${ELASTIC_PRODUCT} && useradd --uid 1000 --gid 1000 --groups 0 --home-dir /usr/share/${ELASTIC_PRODUCT} --no-create-home ${ELASTIC_PRODUCT}
 
 # Elastic Agent permissions
-RUN find /usr/share//elastic-agent/data -type d -exec chmod 0770 {} \; && \
-    find /usr/share//elastic-agent/data -type f -exec chmod 0660 {} \; && \
+RUN find /usr/share//elastic-agent/data -type d -exec chmod 0777 {} \; && \
+    find /usr/share//elastic-agent/data -type f -exec chmod 0666 {} \; && \
```

Do we need to change the world permissions as well? Or we can live with the previous permissions, like:
```suggestion
RUN find /usr/share//elastic-agent/data -type d -exec chmod 0770 {} \; && \
    find /usr/share//elastic-agent/data -type f -exec chmod 0660 {} \; && \
```

---

I aligned permissions with what we have in https://github.com/elastic/elastic-agent/blob/main/dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl because it looks like we missed the Ironbank Dockerfile.